Parent/Parent/ParentAppDelegate.swift

```diff
@@ -209,7 +213,11 @@ extension ParentAppDelegate: NativeLoginManagerDelegate {
     func didLogin(_ client: CKIClient) {
         let session = client.authSession
         self.session = session
-        
+
+        if let token = session.token {
+            AppEnvironment.shared.userDidLogin(session: KeychainEntry(accessToken: token, baseURL: session.baseURL, expiresAt: nil, locale: "en", masquerader: nil, refreshToken: nil, userAvatarURL: session.user.avatarURL, userID: session.user.id, userName: session.user.name))
```

Comment:
`CKIClient.keychainEntry` was defined in an extension in Student and Teacher. It might be good to keep some consistency by doing the same here.

Reply:
I'm not sure if I follow. This `KeychainEntry` is defined in core where it's used by the `AppEnvironment`. It's not using `CKIClient.keychainEntry`.

Reply:
right, Teacher and Student both link to both Core and Keymaster, they both define an extension on CKIClient to get the keychain entry, instead of handling the CKIClient to KeychainEntry mapping in AppDelegate.